backend/src/pong/pong.gateway.ts

```diff
@@ -0,0 +1,69 @@
+import {
+	MessageBody,
+	OnGatewayConnection,
+	SubscribeMessage,
+	WebSocketGateway,
+	WebSocketServer,
+} from '@nestjs/websockets';
+import { Server, Socket } from 'socket.io';
+import { Ball, Paddle } from './Game.types';
+
+@WebSocketGateway({
+	namespace: '/pong',
+	cors: {
+		origin: '*:*',
```

Comment:
Why `*:*` and not just `*`?
I can't find anything about what the extra `:*` would do.

Reply:
Honestly, I have no idea anymore, it was something from a CORS issue on StackOF , but cant remember what is was, was put here out of habit. changed it to "*"